light-client/src/store.rs

```diff
@@ -20,15 +20,38 @@ pub mod sled;
 pub trait LightStore: std::fmt::Debug + Send {
     /// Get the light block at the given height with the given status, or return `None` otherwise.
     fn get(&self, height: Height, status: Status) -> Option<LightBlock>;
+
     /// Update the `status` of the given `light_block`.
     fn update(&mut self, light_block: &LightBlock, status: Status);
+
     /// Insert a new light block in the store with the given status.
     /// Overrides any other block with the same height and status.
     fn insert(&mut self, light_block: LightBlock, status: Status);
+
     /// Remove the light block with the given height and status, if any.
     fn remove(&mut self, height: Height, status: Status);
-    /// Get the highest light block with the given status.
-    fn highest(&self, status: Status) -> Option<LightBlock>;
+
+    /// Get the latest light block with the given status.
+    fn latest(&self, status: Status) -> Option<LightBlock>;
```

Comment:
Just to remove any remaining ambiguity: previously this was called "highest" and it was pretty clear that this refers to the height. Latest is a bit unclear: latest added? Latest as in newest regarding block time, or, height? I think the doc comment could be explicit (the name is pr. fine though).

Reply:
Good point! As in the spec, it means latest as in newest regarding height, will clarify in doc comment.

Reply:
See https://github.com/informalsystems/tendermint-rs/pull/407/commits/dc63808b6f7b89ee37b7d6b837623757ea9ebda9